docs/user/alerting/action-variables.asciidoc

```diff
@@ -1,8 +1,12 @@
 [[rule-action-variables]]
 == Rule action variables
 
-Alerting rules can use the https://mustache.github.io/[Mustache] template syntax
-(`{{variable name}}`) to pass values when its actions run.
+Alerting rules can use the https://mustache.github.io/mustache.5.html[Mustache] template syntax
+(`{{variable name}}`) to pass values when its actions run.  Note that "partials" are not supported.
```

Comment:
I added that, as the mustache docs reference it, and we don't support them (yet, anyway).  I suspect most customers won't know what they are.  Also I'm waiting for the day a customer asks us how to **use** partials, because they saw it in the doc, and we have to tell them we don't support them.

Since that day hasn't come yet, feels fine to remove it.

Reply:
Okay! I've removed that sentence via a437eef26c1613a0968ddfeaf46c1fc23e31c753